src/AlpineSkiHouse.Web/wwwroot/_references.js

```diff
@@ -1,4 +1,9 @@
 ﻿/// <autosync enabled="true" />
+/// <reference path="../bower_components/bootstrap/dist/js/bootstrap.js" />
```

Comment:
I thought that this file is used for typescript, I don't think including it is a problem.